index.js

```diff
@@ -32,7 +32,12 @@ async function main (b, args) {
 
 /* istanbul ignore if */
 if (require.main === module) {
-	main(bin, process.argv.slice(2));
+	main(bin, process.argv.slice(2))
+	  .catch((e) => {
+		  console.error(`An error occurred: ${e}`);
+		  process.exit(1)
+	  })
+	  .then(() => process.exit(0));
```

Comment:
Shouldn't we return the exitCode from underlying execution ?

Reply:
Nvm, the exit is happening already in the `main()` func.

Reply:
@FriggaHel there is no exit code from the underlying execution. If the call to `main` succeeds, then it returns "undefined".

Technically, adding the `.then(() => process.exit(0)` is unnecessary, because if we just let the function call run and then it succeeds, the NodeJS process will exit with 0 by default. This just makes it more explicit.